packages/vitest/src/node/reporters/reported-tasks.ts

```diff
@@ -500,7 +500,12 @@ export interface TaskOptions {
   readonly fails: boolean | undefined
   readonly concurrent: boolean | undefined
   readonly shuffle: boolean | undefined
-  readonly retry: number | undefined
+  readonly retry: number | {
+    count?: number
+    delay?: number
+    condition?: string | ((error: Error) => boolean)
```

Comment:
I don't think this is possible. Please add a test for this to verify how `retry` options are passed down.